boards/nucleo-wl55jc/doc.txt

```diff
@@ -0,0 +1,52 @@
+/**
+@defgroup    boards_nucleo-wl55jc STM32 Nucleo-wl55jc
+@ingroup     boards_common_nucleo64
+@brief       Support for the STM32 Nucleo-wl55jc
+
+### MCU
+
+
+| MCU        |   STM32WL55JC                                             |
+|:---------- |:--------------------------------------------------------- |
+| Family     | ARM Cortex-M4 / ARM Cortex-M0+ Dual core                  |
+| Vendor     | ST Microelectronics                                       |
+| RAM        | 64KiB                                                     |
+| Flash      | 256KiB                                                    |
+| Frequency  | up to 48MHz                                               |
+| FPU        | no                                                        |
+| Timers     | 10 (2x watchdog, 1 SysTick, 6x 16-bit, 1x 32-bit [TIM2])  |
+| ADCs       | 1x 12-bit (12 channel )                                   |
+| UARTs      | 3 (two UARTs and one Low-Power UART)                      |
+| SPIs       | 2                                                         |
+| I2Cs       | 3                                                         |
```

Comment:
Are we saying how many the chip has or how many are available in RIOT? (I don't know).  As I count it there is 1.

Reply:
The chip has 3 and the board does provide access to the three. This document is about the board and not RIOT , right ?